### ISSUE TITLE
Add Chrome Status Feature ID as Metadata

### ISSUE THREAD

**Issue by jpmedley** [google/WebFundamentals#4769]
Add the Chrome status feature ID as metadata, then we could automate adding article links to Chrome status features.

For example, this tag would be picked up by a script:

`{# wf_status_id: 5121613552943104 #}`

And the page it's found in would have it's link deposited in the identified Chrome Status entry. (in this case: https://www.chromestatus.com/feature/5121613552943104).

**Comment by ebidel** [comment]
@petele and I discussed this as well, I can't remember where we landed. I think he was against adding additional metadata for feature id, but I like it :)

Proposal:

```
chromestatus_feautre_id: 5121613552943104
```

**Comment by jpmedley** [comment]
What's the concern with another metadata tag?

**Comment by petele** [comment]
Adding a new metatag is fine. I would rather it be text based though, not a number. Otherwise I'll have to look it up every single time I need something. If I can at least guess and then have a test to verify I'm right, more people will do it properly.

**Comment by ebidel** [comment]
Not sure what you mean by "Otherwise I'll have to look it up every single time I need something.".
You mean you want to validate the id?

**Comment by petele** [comment]
Yes, the IDs should be validated, but they should also be plain text vs integer. For example, I want to include `chromestatus_feautre_id: Blink>HTML>Dialog` instead of `chrome_status_feature_id: 5121613552943104`

**Comment by jpmedley** [comment]
I'd have to cut and paste either way. I'm already in Chrome Status to get the Blink component. I think the cost is negligible and what I get in return, automated links to articles, has a high value to communicating our message.

**Comment by ebidel** [comment]
@petele I think we're talking about two different things. Feature ids map to chromestatus db entries. For example, https://www.chromestatus.com/feature/5761503818940416. "5761503818940416" is the feature id. 

These are different than adding metadata for blink components: https://github.com/google/WebFundamentals/issues/4767

**Comment by ebidel** [comment]
Any objections to supporting a `{# wf_chromestatus_id: 5121613552943104 #}` meta tag on content?

**Comment by petele** [comment]
Is there any way to use something human readable? We're already asking folks to add a bunch of meta that is a bit of a pain, but having one that is completely unguessable and not able to be human consumed, I'm less a fan of.

**Comment by ebidel** [comment]
What about full urls? e.g. https://www.chromestatus.com/features/5121613552943104

**Comment by petele** [comment]
Same thing. I can't look at it and immediately know what it is, or guess what it is. I have to look it up somewhere, which is going to make it harder for people to add, thus it's not going to be added as often as we want. It's gotta be easy!

**Comment by ebidel** [comment]
`{# wf_chromestatus_reference_link: https://www.chromestatus.com/features/5121613552943104 #}` isn't obvious? I don't think it's too hard for folks to figure out that it's a link to related content, especially if we document how/when to add these to new articles.

**Comment by petele** [comment]
Not it's not obvious. I still have to click a link to see what it is. If I'm writing a new article that I think might be similar, I still have to double check. I want to be able to see "Dialog - something" or something similar like that.

**Comment by ebidel** [comment]
I'm not sure we can reliably make that work. Things like feature names can change but the db ids are always permanent.
